sw/device/silicon_creator/rom/e2e/rom_e2e_bootstrap_entry/BUILD

```diff
@@ -48,24 +50,24 @@ package(default_visibility = ["//visibility:public"])
 ]
 
 [
-    opentitan_functest(
+    opentitan_test(
         name = "e2e_bootstrap_entry_{}".format(lc_state),
         cw310 = cw310_params(
+            binaries = {"//sw/device/silicon_creator/rom/e2e:new_empty_test_slot_a": "firmware"},
             bitstream = ":bitstream_e2e_bootstrap_entry_{}".format(lc_state),
             tags = maybe_skip_in_ci(lc_state_val),
-            test_cmds = [
-                "--bitstream=\"$(location {bitstream})\"",
-                "--bootstrap=\"$(location {flash})\"",
-            ],
+            test_cmd = """
+                --bootstrap="{firmware}"
```

Comment:
I would have expected the `--bitstream={bitstream}` to remain necessary. My understanding is that it is not added by the new `cw310_params`.

Reply:
Good catch. Added back.